start/startUnifyingAsynchronicity.txt

```diff
@@ -5,7 +5,7 @@
 
 One of the most serious issues you face when you're programming iOS with Cocoa Touch is the asynchronous event-driven nature of iOS programming. Signals come to you in different places at different times, and your app has to be ready to receive them and cope with them in a coherent way. In a view controller with a table view, you might implement `viewDidLoad`, `viewDidAppear`, `supportedInterfaceOrientations`, `tableView(_:cellForRowAt:)`, `deinit`, and many others, just to cope with the fact that you get different signals telling you about different events at different times.
 
-The issue is particularly severe with regard to signals that affect the _state_ of your application. For example, your application is probably maintaining some data. If that data needs to change in response to a signal, and if you don't manage that change coherently, you can end up with bad data and your app will behave incorrectly. Moreover, you typically have interface whose job is to reflect your data. You have to keep your interface synchronized with the data. If the data gets into a bad state or the synchronization between data and interface breaks down, your app will look wrong.
+The issue is particularly severe with regard to signals that affect the _state_ of your application. For example, your application is probably maintaining some data. If that data needs to change in response to a signal, and if you don't manage that change coherently, you can end up with bad data and your app will behave incorrectly. Moreover, you typically have an interface whose job is to reflect your data. You have to keep your interface synchronized with the data. If the data gets into a bad state or the synchronization between data and interface breaks down, your app will look wrong.
```

Comment:
the change here is to "Moreover, you typically have interface..." - I added "an" to it, as in "an interface"